app/src/app/services/connectors/authentication/authentication.service.ts

```diff
@@ -102,6 +102,29 @@ export class AuthenticationService extends ApiConnector {
         )
     }
 
+    /**
+     * User register sequence
+     * @returns of the logged in user account
+     */
+    public register(): Observable<any> {
+        return this.getAuthType().pipe(
+            // retrieve authentication configuration
+            concatMap(authnType => {
+                let url = `${this.baseUrl}/authn/${authnType}/register`;
+                if (authnType == "oidc") {
+                    // oidc login
+                    url += `?destination=${encodeURIComponent(window.location.href)}`;
```

Comment:
When registering, this throws a Not Found error. It appears the endpoint URL for registration changed: `api/authn/oidc/register` -> `api/user-accounts/register`